programs/ssl/ssl_client2.c

```diff
@@ -198,7 +200,11 @@ int main( void )
 #define USAGE_DTLS \
     "    dtls=%%d             default: 0 (TLS)\n"                           \
     "    hs_timeout=%%d-%%d    default: (library default: 1000-60000)\n"    \
-    "                        range of DTLS handshake timeouts in millisecs\n"
+    "                        range of DTLS handshake timeouts in millisecs\n" \
+    "    mtu=%%d              default: (library default: unlimited)\n"  \
```

Comment:
inconsistent spaces

Reply:
I believe those spaces are actually consistent once the `%%` has been turned into a single `%`, that is when the usage message is actually printed out to the user.